modules/dynamodb/main.tf

```diff
@@ -37,6 +40,18 @@ resource "aws_dynamodb_table" "dynamodb-table" {
   server_side_encryption {
     enabled = var.encryption_enabled
   }
+
+  dynamic "global_secondary_index" {
+    for_each = var.indices
+    content {
+      hash_key           = global_secondary_index.value.hash_key
+      name               = global_secondary_index.value.name
+      read_capacity      = 1
+      write_capacity     = 1
+      projection_type    = global_secondary_index.value.projection_type
```

Comment:
Burde den ikke fått en av følgende verdier: `ALL`, `INCLUDE` eller `KEYS_ONLY`?

Reply:
Jo. Og det er den som bruker modulen sitt ansvar å spesifisere en av de to.